tests/storage/test_receipts.py

```diff
@@ -290,3 +290,76 @@ def test_get_last_receipt_event_id_for_user(self) -> None:
             )
         )
         self.assertEqual(res, event2_1_id)
+
+    def test_receipts_clear_push_actions(self) -> None:
+        event_1_id = self.create_and_send_event(
+            self.room_id1, UserID.from_string(OTHER_USER_ID)
+        )
+        event_2_id = self.create_and_send_event(
+            self.room_id1, UserID.from_string(OTHER_USER_ID)
+        )
+
+        def _assert_push_action_count(expected: int):
+            result = self.get_success(
+                self.store.db_pool.simple_select_list(
+                    table="event_push_actions",
+                    keyvalues={"1": 1},
```

Comment:
You can also just set `keyvalues` to `None` which generates an SQL query with no `WHERE` clause